hapi-fhir-base/src/main/java/ca/uhn/fhir/rest/gclient/IRawHttp.java

```diff
@@ -0,0 +1,10 @@
+package ca.uhn.fhir.rest.gclient;
+
+/**
+ * Interface to make non-FHIR requests using the GenericClient plumbing.
+ * Useful for the interceptor infrastructure when dealing with thinks like $export poll locations
+ * which are not FHIR payloads.
+ */
+public interface IRawHttp {
+	IClientHttpExecutable<IClientHttpExecutable<?, IEntityResult>, IEntityResult> get(String theUrl);
```

Comment:
public interface method should have javadoc
```suggestion
	/**
	 * Creates a GET request to the specified URL.
	 *
	 * @param theUrl The URL to send the GET request to. Can be absolute or relative
	 *               to the client's base URL.
	 * @return A builder for configuring and executing the HTTP GET request
	 */
	IClientHttpExecutable<IClientHttpExecutable<?, IEntityResult>, IEntityResult> get(String theUrl);
```